ml-agents/README.md

```diff
@@ -1,5 +1,22 @@
-# Python API
+# Unity ml-agents interface and trainers
 
+The `mlagents` package contains two components : The low level API which allows you to interact directly with a Unity Environment and a training component whcih allows you to train agents in Unity Environments using our implementations of reinforcement learning or imitation learning.
+
+## Installation
+
+The `ml-agents` package can be installed using:
+
+```
+pip install mlagents
+```
+
+or by running the following from the `ml-agents` directory of the repository:
+
+```
+pip install .
+```
+
+## Low level API
```

Comment:
If we call this the "low level API" should we refer to the training api as the high level? Maybe just be explicit that this is `mlagents.envs` and the other is `mlagents.trainers`.